_unit-test/create-docker-volumes-test.sh

```diff
@@ -15,14 +15,13 @@ sentry-kafka
 sentry-postgres
 sentry-redis
 sentry-symbolicator
+sentry-vroom
```

Comment:
I feel like this is not a great practice (creating and relying on global volumes). Can we not make this a "local" volume? Do we need to absolutely make sure the data here should survive a potentially accidental `docker compose down -v`?

Reply:
This will end up with profile data, it's something you'd likely want persisted by default.

Reply:
Local volumes are also persisted by default. They are just not "globally" available by docker containers outside of this project and you can remove them with `docker compose down -v`.

I'd say we should stop adding new global volumes and keep everything local unless we have _very_ good reasons and we understand them fully.

Reply:
Profiling data should survive accidental `docker compose down`, though. Although AFAIK, [vroom](https://github.com/getsentry/vroom) doesn't have a cleanup mechanism for profiling data that exceeds the maximum sentry retention period. I guess for now, every profiling data stored on this volume will be persisted forever.

Reply:
Switching from global to local volumes seems like a reasonable improvement, but I don't think we should make @phacops implement that for us. Let's let him use the status quo patterns and make the switch in a separate PR.

Reply:
@chadwhitacre this one does not require a refactor. Just not adding it here and removing the `external: true` line in the compose file will do it. We already have other local volumes.

Reply:
@BYK It's definitely an issue we didn't really think about since we were relying on GCS retention settings to remove profiles when needed. I should be addressed separately in `vroom`.

Reply:
@BYK Now, the reason why I set the `external: true` was because I wanted the volume to be persisted and according to the comment we have for non-external volumes, it suggests they won't be persisted. Seems like the comment is misguided though.